ci/kubetest/test_clickhouse_persistence_disabled.py

```diff
@@ -15,14 +15,13 @@
 """
 
 
-@pytest.fixture
-def setup(kube):
+def test_helm_install(setup, kube):
     cleanup_k8s()
     helm_install(HELM_INSTALL_CMD)
     wait_for_pods_to_be_ready(kube)
 
 
-def test_volume_claim(setup, kube):
+def test_volume_claim(kube):
```

Comment:
Does this mean we rely on `test_helm_install` having run prior to this one?

Reply:
Yes

Reply:
I think we should just drop the non-hpa test changes and get this in, is that possible?

I think test interdependence is something we should avoid, as will make it hard to e.g. like you said introduce parallelism among other things